capa/ida/plugin/view.py

```diff
@@ -1029,7 +1023,7 @@ def parse_features_for_tree(self, parent, features):
 
         def format_address(e):
             if isinstance(e, AbsoluteVirtualAddress):
-                return "%X" % int(e)
+                return f"{hex(int(e))}"
```

Comment:
this should be good, but we should test the IDA plugin and standalone output on a few samples manually, as well